Dockerfile

```diff
@@ -1,18 +1,20 @@
 FROM python:3.6.3
 
-# Create an unprivileged user
+# Set up a workspace
 ENV PSHTT_HOME=/home/pshtt
-RUN mkdir ${PSHTT_HOME} 
-RUN groupadd --system pshtt \
-    && useradd --system --comment="pshtt user" --gid="pshtt" pshtt
+RUN mkdir ${PSHTT_HOME}
+
+# Create an unprivileged user
+RUN groupadd --system pshtt
```

Comment:
@egyptiankarim, do you mind combining these two `RUN` statements like they were originally?  I'm kind of insane about combining related statements where possible to reduce the number of layers in the Docker image.